src/Model/Asset/AssetOptions.php

```diff
@@ -92,12 +93,10 @@ public function toArray(): array
     public function toBytes(): string
     {
         return implode('', [
-            str_pad(Math::gmpDecHex(Math::reverseBytesLong($this->getMaxSupply())), 16, '0', STR_PAD_LEFT),
+            Math::getInt64($this->getMaxSupply()),
             $this->getExchangeRate()->toBytes(),
             $this->isExchangeable() ? '01' : '00',
-            str_pad(count($this->getExtensions()), 2, '0', STR_PAD_LEFT),
-            // TODO: isFixedMaxSupply
-//            '01'
```

Comment:
unresolved todo

Reply:
This is a deleted code, aka has been fixed and this section doesn't exist anymore.